app/controllers/tagging_history_controller.rb

```diff
@@ -0,0 +1,26 @@
+class TaggingHistoryController < ApplicationController
+  before_action :ensure_user_can_access_tagathon_tools!
+
+  def index
+    render :index, locals: { page: TaggingHistory::IndexPage.new({ link_types: ['taxons'] }.merge(filter_params)) }
```

Comment:
link_types: ['taxons'] could be merged in the IndexPage itself, so this bit becomes simpler

Reply:
I've now done this as part of refactoring this class in f93598b